docs/en/connector-v2/source/Hive.md

```diff
@@ -111,6 +103,25 @@ Source plugin common parameters, please refer to [Source Common Options](common-
 
 ```
 
+### Example 2: Multiple tables
+
+```bash
+
+  Hive {
+    tables_configs = [
```

Comment:
A small suggestion is that this is better named using `table_list`, maximizing uniform parameter naming

Reply:
Use `tables_configs` is consistent with other file connectors.